src/site/docs/tutorials/get-started/index.markdown

```diff
@@ -515,7 +510,7 @@ namely the main() function.
 The two apps you've seen in this tutorial have other top-level functions.
 The Hello World example calls print(),
 a top-level function defined in dart:core.
-And the clickme app defines a top-level function called reverseText().
+And the simple app calls querySelector().text to print text in the browser.
```

Comment:
-> And the simple app calls querySelector(), a top-level function defined in dart:html.